pset-functions.js

```diff
@@ -30,7 +30,34 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+function blackJack (playerCardScore, dealerCardScore) {
+  //  first, let's determine if there are any entries above 21
+  if ((playerCardScore > 21) && (dealerCardScore > 21)) {
+    return 0
+  } else if (playerCardScore > 21) {
+    return dealerCardScore
+  } else if (dealerCardScore > 21) {
+    return playerCardScore
+  } else {
+    //  if none of the entries are above 21 the score will be calculated
+    calculateScore(playerCardScore, dealerCardScore)
+  }
+}
+
+function calculateScore (player, dealer) {
+  //  calculate the difference from 21 for each score
+  const playerScore = 21 - player
+  const dealerScore = 21 - dealer
+
+  //  if playerScore has a larger difference from 21 then return dealerScore
+  if (playerScore > dealerScore) {
+    return dealerScore
+  } else {
+    return playerScore
+  }
+}
+
+console.log(blackJack(19, 22))
```

Comment:
@sad3xoxo Good work here on this problem I see you have your blackjack function checking to see the player's values and you have the calculate score to return the score. 
For `calculateScore(playerCardScore, dealerCardScore)` since that function is returning a value you need to include the keyword `return`. the `return` will allow you to get the value from the `calculateScore` function and display is you when you `console.log(blackJack(19, 22))` on line 60. 
One suggestion for your `calculateScore` function currently you're returning the `playerScore/dealerScore` after you have subtracted 21 - `player/dealer`. You could return the player or dealer value in the conditional statement.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)